src/transport/UnauthenticatedSessionTable.h

```diff
@@ -87,7 +90,7 @@ class UnauthenticatedSession : public Session,
         switch (mPeerAddress.GetTransportType())
         {
         case Transport::Type::kUdp:
-            return GetRemoteMRPConfig().mIdleRetransTimeout * (CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS + 1);
+            return IsPeerActive() ? mRemoteMRPActiveTimeout : mRemoteMRPIdleTimeout;
```

Comment:
Again, do we have to store the values?  Seems like we could easily recompute them here.